cloudbuild.yaml

```diff
@@ -365,11 +373,11 @@ steps:
           exit 1
         fi
 
-        # TODO: re-add this check once generating embeddings in CI is stable
-        # if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
-        #   echo "rag frontend test failed"
-        #   exit 1
-        # fi
+        TODO: re-add this check once generating embeddings in CI is stable
```

Comment:
remove this TODO since it's no longer a comment

Reply:
Added the PR with this change.